sdcm/cluster.py

```diff
@@ -2932,6 +2932,47 @@ def get_nic_devices(self) -> List:
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         return result.stdout.strip().split()
 
+    def run_scylla_sysconfig_setup(self) -> bool:
+        """
+        Run the scylla_sysconfig_setup script that
+        sets the values in /etc/scylla.d for:
+        - cpuset.conf
+        - perftune.yaml
+        These values are used by Scylla to decide
+        on how to balance rx queues with the logical
+        core number.
+
+        Returns a bool indicating if a restart is
+        required if the previous values
+        of cpuset.conf are different from the
+        new ones. Note: restarting might trigger a
+        resharding in Scylla.
+        """
+        # backup old config if it exists
+        self.log.info("Move current config files before running scylla_sysconfig setup...")
+        self.remoter.sudo("mv /etc/scylla.d/cpuset.conf /etc/scylla.d/cpuset.conf.old", ignore_status=True)
+        self.remoter.sudo("mv /etc/scylla.d/perftune.yaml /etc/scylla.d/perftune.yaml.old", ignore_status=True)
+
+        # run as sudo scylla_sysconfig_setup
+        self.log.info("Running scylla_sysconfig_setup as sudo...")
+        nic_name = self.get_nic_devices()[0]
+        self.remoter.sudo(f"scylla_sysconfig_setup --nic {nic_name} --homedir /var/lib/scylla --confdir /etc/scylla")
+
+        # compare old output witn new in scylla.d
+        self.log.info("Comparing old config files vs new ones after running scylla_sysconfig_setup...")
+        cpuset_diff = self.remoter.run(
+            "diff -q /etc/scylla.d/cpuset.conf /etc/scylla.d/cpuset.conf.old", ignore_status=True)
+
+        self.log.info("Comparison result:\nstdout: %s\nstderr: %s", cpuset_diff.stdout, cpuset_diff.stderr)
```

Comment:
IMHO it should be `debug`

Reply:
Most of these log messages should be in debug, but I'd leave these only :`running scylla sysconfig setup` and add log message that restart is required/not required due change/no change in cpuset/perftune.yaml file.

Reply:
I'll reduce the number and log level of them after I'm done with running tests for this PR. Currently some of them are just for debug purposes.

Reply:
I wouldn't change to debug or reduce the number of them.
Later when someone try to understand what happens here it's very useful.
It's not something that runs very often or produce too much output to consider it as a problem.

Reply:
The fact that you needed them for your debug process, means that once someone will need these messages to debug an issue here.